code/modules/cm_marines/overwatch.dm

```diff
@@ -1030,7 +1030,10 @@ GLOBAL_LIST_EMPTY_TYPED(active_overwatch_consoles, /obj/structure/machinery/comp
 				log_game("[key_name(user)] set red alert via the groundside operations console.")
 				message_admins("[key_name_admin(user)] set red alert via the groundside operations console.")
 				log_ares_security("Manual Security Update", "Changed the security level to red.", user)
-				set_security_level(SEC_LEVEL_RED, no_sound = TRUE, announce = FALSE)
+				if(!COOLDOWN_FINISHED(datacore, ares_quarters_cooldown))
+					set_security_level(SEC_LEVEL_RED, no_sound = FALSE, announce = TRUE)
+				else
+					set_security_level(SEC_LEVEL_RED, no_sound = TRUE, announce = FALSE)
```

Comment:
Rather than having it still change to red without using the swipers, just move the setting to below the existing check.